### PR TITLE
Fix/logo max size twitter tiles etc [Delivers #114257721]

### PR DESCRIPTION
**Pull request by chiefGui**
@chute/frontend

### CODE REVIEW

File: src/components/live-displays/Grid/GridAsset.jsx [hunk 1]
```diff
@@ -42,14 +42,15 @@ export default class GridAsset extends Component {
...
       </div>
     )
   }
 
-  renderAsset() {
+  renderAsset(computedStyle = {}) {
     const {asset} = this.props
-    const style = {backgroundImage: `url("${this.props.asset.url}")`}
+    const naturalBackgroundColor = {backgroundImage: `url("${this.props.asset.url}")`}
+    const style = Object.assign(computedStyle, naturalBackgroundColor)
```
Comment (jonathan-potter) [inline src/components/live-displays/Grid/GridAsset.jsx]: i don't really understand the name `naturalBackgroundColor`. the variable is a url for an image. Is the image a solid color?

a better name would describe what the variable actually is. Is it the default background image?

File: src/components/live-displays/Grid/GridAsset.scss [hunk 2]
```diff
@@ -98,7 +98,9 @@ $column2-left-position: $asset--normal-width * 2;
   left:   initial;
 
   width:  50.6%;
-  height: 100%;
+  height: calc(100% - 2px); // calculating based in the border's size
```
Comment (jonathan-potter) [inline src/components/live-displays/Grid/GridAsset.scss]: `box-sizing: border-box;`?

**Comment by chiefGui** [comment]
Once again, @chute/frontend ?

**Comment by evandrodutra** [comment]
:+1:

**Comment by diogotrentini** [comment]
:+1:

**Pull request merged**